pom.xml

```diff
@@ -100,7 +100,7 @@
     <!-- Exclude all generated code -->
     <sonar.exclusions>**/generated-sources</sonar.exclusions>
 
-    <plc4x-code-generation.version>1.4.0</plc4x-code-generation.version>
+    <plc4x-code-generation.version>1.5.0-SNAPSHOT</plc4x-code-generation.version>
```

Comment:
uh are there some new upstream feature required?

Reply:
Yes there is an feature/native_opcua branch for the build-utils that includes some changes to the string type references.

Reply:
@chrisdutz do we need to release the code-generation stuff first or can we leave it at SNAPSHOT